course_inventory/inventory.py

```diff
@@ -65,6 +67,57 @@ def make_request_using_api_utils(url: str, params: Dict[str, Any] = {}) -> Respo
     return response
 
 
+def gather_new_term_data_from_api(
+    account_id: int,
+    term_ids: Sequence[int],
+    creator_obj: DBCreator
+) -> pd.DataFrame:
```

Comment:
New style of putting each function argument on a separate line?

Reply:
If it gets too long, why not?

Reply:
I agree.  I asked because there are other function definitions you changed in this PR that could also have their arguments arranged on separate lines, but you didn't do that.  E.g.,…

* https://github.com/tl-its-umich-edu/course-inventory/pull/115/files#diff-3ab4d3be8a800981ae3e09e15194bc0dR142
* https://github.com/tl-its-umich-edu/course-inventory/pull/115/files#diff-9e1a06acd700cfb38555dc5980f1cdb0R163

I know that, "A foolish consistency is the hobgoblin of little minds".  I hope it's not foolish to say this style stands out (in a good way) and I wondered whether it should apply in these other instances, too.

Reply:
Okay, I changed the latter one. The former has only two parameters and isn't nearly as long, so I'm going to pass on that one, it that's acceptable.

Reply:
Also, I changed the term function a bit, so it doesn't need the `creator_obj` anymore, meaning I think I can push this back to one line.

Reply:
Resolving this for now.